src/plone/testing/_z2_testbrowser.py

```diff
@@ -1,16 +1,15 @@
 # -*- coding: utf-8 -*-
 from cStringIO import StringIO
+from zope.testbrowser.testing import PublisherConnection
+from zope.testbrowser.testing import PublisherResponse
+
 import base64
 import mechanize
 import re
 import rfc822
 import sys
 import urllib
 import urllib2
-
-# Using a from-import here to avoid an AttributeError below when using
-# zope.testbrowser 4.x without zope.app.testing:
-from zope.testbrowser.testing import PublisherConnection, PublisherResponse
```

Comment:
Okay, I found out that we are using `_z2_testbrowser.py` together with `zope.app.testing` in _union.cms_ but we have the chance there to switch to WSGI even for the tests as the application already runs on WSGI.